vllm/config/vllm.py

```diff
@@ -12,6 +12,7 @@
 from contextlib import contextmanager
 from dataclasses import replace
 from datetime import datetime
+from enum import Enum
```

Comment:
Means you don't have to use `.value` for integer comparisons

```suggestion
from enum import IntEnum
```